core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.spec.ts

```diff
@@ -88,6 +88,29 @@ const archivedExperiments: DotExperiment[] = [
         goals: null
     }
 ];
+
+const runningExperiments: DotExperiment[] = [
```

Comment:
why don't use the current mocks and just overwrite the property in the test ?

Reply:
Updated.